mercury/views/simulator.py

```diff
@@ -4,21 +4,70 @@
 
 # from ..forms import VehicleForm, SuspensionForm
 from ..forms import SimulatorForm
+from mercury.models import SimulatedData
 import datetime
 
-# from mercury.models import Vehicle,SuspensionSensor
-
 
+# Modified by: Rajeev
+# User story: #95 Continuous Submission for simulator UI
+# begin
 class SimulatorView(TemplateView):
     template_name = "simulator.html"
     form_class = SimulatorForm
 
     def post(self, request, *args, **kwargs):
-        form = self.form_class(request.POST)
-        if form.is_valid():
-            form.save()
-            return HttpResponseRedirect("/simulator/")
-        return render(request, self.template_name, {"form": form})
+        post_name = request.POST.get("the_name")
```

Comment:
Is there any way to not have to manually assign every one of these, or is it required?

Reply:
for sure there will be a way but I know only to do manually, don't have much time to do it in the best way

Reply:
That's fine for now then